<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wMenu</class>
 <widget class="QMainWindow" name="wMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnShowLinear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Linear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMultivariable">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Multi-Variable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGA">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Genetic Algorithm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConstrained">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Constrained</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMultiobjective">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Multi-Objective</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWing">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Wing Question</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnManufacturing">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Manufacturing Question</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
